Sources/Harmony/Data/DataSource/GetNetworkDataSource.swift

```diff
@@ -0,0 +1,84 @@
+//
+// Copyright 2022 Mobile Jazz SL
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Alamofire
+import Foundation
+
+public class GetNetworkDataSource<T: Decodable>: GetDataSource {
+    
+    private let url: URL
+    private let session: Session
+    private let decoder: JSONDecoder
+    
+    public init(url: URL, session: Session, decoder: JSONDecoder) {
+        self.url = url
+        self.session = session
+        self.decoder = decoder
+    }
+    
+    @discardableResult
+    open func getAll(_ query: Query) -> Future<[T]> {
+        return execute(query)
+    }
+    
+    @discardableResult
```

Comment:
why get and getAll are discardable result?

Reply:
I don't know why it got added, I don't see a need for it. Removed